meta-lmp-bsp/dynamic-layers/meta-ti-bsp/recipes-kernel/linux/linux-lmp-ti-staging_6.6.bb

```diff
@@ -25,3 +25,20 @@ module_conf_rpmsg_client_sample = "blacklist rpmsg_client_sample"
 module_conf_ti_k3_r5_remoteproc = "softdep ti_k3_r5_remoteproc pre: virtio_rpmsg_bus"
 module_conf_ti_k3_dsp_remoteproc = "softdep ti_k3_dsp_remoteproc pre: virtio_rpmsg_bus"
 KERNEL_MODULE_PROBECONF += "rpmsg_client_sample ti_k3_r5_remoteproc ti_k3_dsp_remoteproc"
+
+# not found in linux-lmp-ti-staging-6.6.44
+# meta-ti-bsp/conf/machine/am62xx-evm.conf
+# meta-ti-bsp/conf/machine/beagleplay.conf
+KERNEL_DEVICETREE:remove:am62xx = " \
+    ti/k3-am625-phyboard-lyra-1-4-ghz-opp.dtbo \
+"
+
+# not found in linux-lmp-ti-staging-6.6.44
+# meta-ti-bsp/conf/machine/include/am64xx.inc
+KERNEL_DEVICETREE:remove:am64xx = " \
+    ti/k3-am642-hummingboard-t-pcie.dtbo \
+    ti/k3-am642-hummingboard-t-usb3.dtbo \
+    ti/k3-am642-hummingboard-t.dtb \
+    ti/k3-am642-phyboard-electra-gpio-fan.dtbo \
+    ti/k3-am642-phyboard-electra-pcie-usb2.dtbo \
```

Comment:
I don't understand why the machine from the same BSP layer is not including dtbs from the BSP specific kernel tree, both should be aligned here.

Reply:
I don't know. I will drop this commit from part 3 and when I create the part 4 I try to figure it out